client/src/components/AddPreview.vue

```diff
@@ -156,7 +156,7 @@ const testVideos = import.meta.env.DEV
 				"test dash 3 w/ multiple captions",
 				"https://livesim2.dashif.org/vod/testpic_2s/multi_subs.mpd",
 			],
-			["test peertube 0", "https://the.jokertv.eu/w/7C5YZTLVudL4FLN4JmVvnA"],
+			["test peertube 0", "https://tube.shanti.cafe/w/96kFpg7fs4LwYkFCMFv51E"],
```

Comment:
I would also accept a change to replace `the.jokertv.eu` in the default config with a different instance :)